website/docs/components/copy/button/partials/code/how-to-use.md

```diff
@@ -0,0 +1,54 @@
+## How to use this component
```

Comment:
If we want to support the `@encoded` argument, we should add an example to explain what it does (and why it may be used)